common/flavorization/wc_title_holders.txt

```diff
@@ -86,128 +127,1953 @@ warlord_female = {
-	governments = { feudal_government clan_government }
 }
-#Count
-overlord_male = {
+
+### High Elves / Blood Elves ###
+# Feudal
+count_feudal_male_highelf = {
 	type = character
 	gender = male
 	special = holder
 	tier = county
-	priority = @county_cultural_prio
+	priority = 400
 	top_liege = no
-	culture_groups = { orc_group }
-	governments = { feudal_government clan_government }
+	governments = { feudal_government }
+	cultures = { high_elf blood_elf }
 }
-overlord_female = {
+count_feudal_female_highelf = {
 	type = character
 	gender = female
 	special = holder
 	tier = county
-	priority = @county_cultural_prio
+	priority = 400
 	top_liege = no
-	culture_groups = { orc_group }
-	governments = { feudal_government clan_government }
+	governments = { feudal_government }
+	cultures = { high_elf blood_elf }
 }
-
-### Nightborne
-# Emperor
-king_nightborne_male = {
+county_feudal_highelf = {
+	type = title
+	tier = county
+	priority = 400
+    governments = { feudal_government }
+	cultures = { high_elf blood_elf }
+}
+emperor_feudal_male_highelf = {
 	type = character
 	gender = male
 	special = holder
 	tier = empire
-	priority = @empire_cultural_prio
+	priority = 400
 	top_liege = no
 	governments = { feudal_government }
-	cultures = { nightborne highborne }
+	cultures = { high_elf blood_elf }
 }
-queen_nightborne_female = {
+emperor_feudal_female_highelf = {
 	type = character
 	gender = female
 	special = holder
 	tier = empire
-	priority = @empire_cultural_prio
+	priority = 400
 	top_liege = no
 	governments = { feudal_government }
-	cultures = { nightborne highborne }
+	cultures = { high_elf blood_elf }
+}
+empire_feudal_highelf = {
+	type = title
+	tier = empire
+	priority = 400
+    governments = { feudal_government }
+	cultures = { high_elf blood_elf }
+}
+
+### Highborne / Nightbornes ###
+# Nighborne specific
+count_feudal_male_nightborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne }
+}
+count_feudal_female_nightborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne }
+}
+county_feudal_nightborne = {
+	type = title
+	tier = county
+	priority = @county_cultural_prio
+	cultures = { nightborne }
+    governments = { feudal_government theocracy_government }
 }
 
-# King
-grand_prince_nightborne_male = {
+# Feudal
+duke_feudal_male_highborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 400
+	top_liege = no
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne highborne night_elf }
+}
+duke_feudal_female_highborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 400
+	top_liege = no
+	governments = { feudal_government theocracy_government }
```

Comment:
You prob need to add `clan_government` right after every use of `feudal_government` since otherwise, they will have no flavor.

Reply:
done in 671dd5c74